pkg/apis/componentconfig/types.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
what does "componentconfig" refer to? From the comment at the start of this file I would have expected this to be called "controllerconfig" or something like that.